about/core.html

```diff
@@ -16,107 +16,100 @@ <h1 class="page-header">FAMNM Core Team 2019-2020</h1>
                     <span class="leadership-overlay">
-                        <span class="bold">Director of Internal Affairs</span> &bullet; Amanda Kline
+                        <span class="bold">Director of Internal Affairs</span> &bullet; Birghton Pauli
                         <span style="display: block;">famnm.admin@umich.edu</span>
                     </span>
                 </a></li>
                 <li><a href="mailto:famnm.projects@umich.edu">
-                    <span class="img-contain"><img src="../img/core/projects.jpg" alt="Austin Barkey" title="Austin Barkey"/></span>
+                    <span class="img-contain"><img src="../img/core/projects.jpg" alt="Rebecca Jean" title="Rebecca Jean"/></span>
                     <span class="leadership-overlay">
-                        <span class="bold">Director of Project Development</span> &bullet; Austin&nbsp;Barkey
+                        <span class="bold">Director of Project Development</span> &bullet; Rebecca&nbsp;Jean
                         <span style="display: block;">famnm.projects@umich.edu</span>
                     </span>
                 </a></li>
                 <li><a href="mailto:famnm.corporate@umich.edu">
-                    <span class="img-contain"><img src="../img/core/corporate-nick.png" alt="Nick Eidsmoe" title="Nick Eidsmoe"/></span>
+                    <span class="img-contain"><img src="../img/core/corporate.jpg" alt="Rebecca Kowalczyk" title="Rebecca Kowalczyk"/></span>
                     <span class="leadership-overlay">
-                        <span class="bold">Co-director of Corporate Outreach</span> &bullet; Nick&nbsp;Eidsmoe
-                        <span style="display: block;">famnm.corporate@umich.edu</span>
-                    </span>
-                </a></li>
-                <li><a href="mailto:famnm.corporate@umich.edu">
-                    <span class="img-contain"><img src="../img/core/corporate-michelle.jpg" alt="Michelle Wang" title="Michelle Wang"/></span>
-                    <span class="leadership-overlay">
-                        <span class="bold">Co-director of Corporate Outreach</span> &bullet; Michelle&nbsp;Wang
+                        <span class="bold">Director of Corporate Outreach</span> &bullet; Rebecca&nbsp;Kowalczyk
                         <span style="display: block;">famnm.corporate@umich.edu</span>
                     </span>
                 </a></li>
                 <li><a href="mailto:famnm.treasurer@umich.edu">
-                    <span class="img-contain"><img src="../img/core/treasurer.png" alt="Eashwar Mohan" title="Eashwar Mohan"/></span>
+                    <span class="img-contain"><img src="../img/core/treasurer.jpg" alt="Chan Desai" title="Chan Desai"/></span>
                     <span class="leadership-overlay">
-                        <span class="bold">Treasurer</span> &bullet; Eashwar Mohan
+                        <span class="bold">Treasurer</span> &bullet; Chan Desai
```

Comment:
Were we able to confirm with Chan if he wants his full name on here? (Chandra)

Reply:
yup, still waiting on confirmation so this is in until then as that's what he's got on the ri3d page